examples/example-column-hidden.html

```diff
@@ -0,0 +1,401 @@
+    sortdir = args.sortAsc ? 1 : -1;
+    sortcol = args.sortCol.field;
+
+    if (isIEPreVer9()) {
+      // using temporary Object.prototype.toString override
+      // more limited and does lexicographic sort only by default, but can be much faster
+
+      var percentCompleteValueFn = function () {
+        var val = this["percentComplete"];
+        if (val < 10) {
+          return "00" + val;
+        } else if (val < 100) {
+          return "0" + val;
+        } else {
+          return val;
+        }
+      };
+
+      // use numeric sort of % and lexicographic for everything else
+      dataView.fastSort((sortcol == "percentComplete") ? percentCompleteValueFn : sortcol, args.sortAsc);
+    } else {
+      // using native sort with comparer
+      // preferred method but can be very slow in IE with huge datasets
+      dataView.sort(comparer, args.sortAsc);
+    }
+  });
+
+  // wire up model events to drive the grid
+  // !! both dataView.onRowCountChanged and dataView.onRowsChanged MUST be wired to correctly update the grid
+  // see Issue#91
+  dataView.onRowCountChanged.subscribe(function (e, args) {
+    grid.updateRowCount();
+    grid.render();
+  });
+
+  dataView.onRowsChanged.subscribe(function (e, args) {
+    grid.invalidateRows(args.rows);
+    grid.render();
+  });
+
+  dataView.onPagingInfoChanged.subscribe(function (e, pagingInfo) {
+    grid.updatePagingStatusFromView( pagingInfo );
+
+    // show the pagingInfo but remove the dataView from the object, just for the Cypress E2E test
+    delete pagingInfo.dataView;
+    console.log('on After Paging Info Changed - New Paging:: ', pagingInfo);
+  });
+
+  dataView.onBeforePagingInfoChanged.subscribe(function (e, previousPagingInfo) {
+    // show the previous pagingInfo but remove the dataView from the object, just for the Cypress E2E test
+    delete previousPagingInfo.dataView;
+    console.log('on Before Paging Info Changed - Previous Paging:: ', previousPagingInfo);
+  });
+
+  var h_runfilters = null;
+
+  // wire up the slider to apply the filter to the model
+  var slider = document.getElementById("pcSlider");
+  var slider2 = document.getElementById("pcSlider2");
+  var sliderCallback = function() {
+    Slick.GlobalEditorLock.cancelCurrentEdit();
+
+    if (percentCompleteThreshold != this.value) {
+      window.clearTimeout(h_runfilters);
+      h_runfilters = window.setTimeout(updateFilter, 10);
+      percentCompleteThreshold = this.value;
+    }
+  }
+  
+  slider.oninput = sliderCallback;
+  slider2.oninput = sliderCallback;
+
+  // wire up the search textbox to apply the filter to the model
+  document.querySelector('#txtSearch,#txtSearch2').addEventListener("keyup", function(e) { 
+    Slick.GlobalEditorLock.cancelCurrentEdit();
+
+    // clear on Esc
+    if (e.which == 27) {
+      this.value = "";
+    }
+
+    searchString = this.value;
+    updateFilter();
+ });
+
+//  $("#txtSearch,#txtSearch2").keyup(function (e) {
```

Comment:
again some more code you can probably delete, it goes with the previous one to add filter in the preheader